packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "20.0.0",
+  "version": "22.0.0",
```

Comment:
Is there a reason this needs to be incremented by two major releases?

Reply:
I made this change prior to our meeting on versioning. I have reverted this change to reflect the current version (20.0.0) for the appropriate engineer to update  core's version on release.